sdk/java/runtime/main.go

```diff
@@ -141,7 +142,7 @@ func (m *JavaSdk) buildJavaDependencies(
 	}
 	return ctr.
 		// Cache maven dependencies
-		WithMountedCache("/root/.m2", dag.CacheVolume("sdk-java-maven-m2")).
+		WithMountedCache("/root/.m2", dag.CacheVolume("maven-m2", dagger.CacheVolumeOpts{Namespace: "sdk-java"}), dagger.ContainerWithMountedCacheOpts{Sharing: dagger.CacheSharingModeLocked}).
```

Comment:
I would probably not set the namespace - I actually hadn't realized it was possible to set manually :cry: 

It's intended as a unique value per module (so each calling module gets a unique value of it).